cmd/run.go

```diff
@@ -508,13 +509,14 @@ func readSource(src, pwd string, fs afero.Fs, stdin io.Reader) (*lib.SourceData,
 		if err != nil {
 			return nil, err
 		}
-		return &lib.SourceData{Filename: "-", Data: data}, nil
+		return &lib.SourceData{URL: &url.URL{Path: "-", Scheme: "file"}, Data: data}, nil
 	}
-	abspath := filepath.Join(pwd, src)
-	if ok, _ := afero.Exists(fs, abspath); ok {
-		src = abspath
+	pwdURL := &url.URL{Scheme: "file", Path: filepath.ToSlash(filepath.Clean(pwd))}
```

Comment:
it actually works great with both because of the next line ... the thing it doesnt work great with (which I found out later) is `k6 run script.js` but it works with `k6 run ./script.js` as maybe I will need to return the `afero.Exists` ;(.

Reply:
ah, my mistake, missed that this is just the `pwd`